### PR TITLE
migration and quickstart

### PR DESCRIPTION
**Pull request by kjaym**
@maxhutch This puts clarity into the migration page, adds a quickstart, and changes some folder names.

### CODE REVIEW

File: docs/01_quickstart/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+)
+
+results = client.search(query)
+```
+
+Results are return as a [`PifSearchResult`](!api/search/pif/result/PifSearchResult) object.
+
+### Next steps
+
+Users are encourage to read about the data model used on Citrination, the [PIF](http://citrine.io/pif), in order to understand how data is represented on the system.
```
Comment (maxhutch) [inline docs/01_quickstart/index.md]: encouraged

**Comment by maxhutch** [comment]
Other than a single grammer, this is a huge improvement.

Aside: would it be possible to build PRs and put them online somewhere to make reviewing the markdown easier?

**Comment by kjaym** [comment]
@maxhutch Fixed the typo. 

I don't have a good system in place for doing auto builds. The generate file in this package creates the static deploy files. We have to run that and then move the static files to an S3 bucket or something. It's possible if we want to invest the time in it.

**Comment by maxhutch** [comment]
Could we use github pages?

**Comment by kjaym** [comment]
It uses github pages now. That's fairly automatic -  you push into a gh-pages branch all of the static files. I'm not sure how we would do that in the same repo on a pull request.

**Pull request merged**